src/main/java/de/embl/cba/mobie/source/ImageDataFormat.java

```diff
@@ -0,0 +1,59 @@
+package de.embl.cba.mobie.source;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum ImageDataFormat
+{
+	@SerializedName( "bdv.n5" )
+	BdvN5,
+	@SerializedName( "bdv.n5.s3" )
+	BdvN5S3,
+	@SerializedName( "openOrganelle" )
```

Comment:
I think we should call this one "openOrganelle.s3":
That's in line with he bdv naming schemes and would allow to add a local version of the openOrganelle file format if we want to support this in the future.

Reply:
OK. done.